faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -23,6 +29,7 @@ object Decoder {
     def decode(v: Value, path: FieldPath) =
       v match {
         case StringV(s) => Result.successful(s, path)
+        case b: BytesV => Result.successful(b.strValue, path)
```

Comment:
Do we have any implicit from `String` to `BytesV`?

IMO, implicits are a nice syntax sugar but, in this case, I would prefer to be strict about the types people are getting in and out of the database so we avoid wrong assumptions about the underneath schema.